src/command/check/check.c

```diff
@@ -178,53 +182,58 @@ cmdCheck(void)
 
     MEM_CONTEXT_TEMP_BEGIN()
     {
-        // Build stanza list based on whether a stanza was specified or not
-        StringList *stanzaList;
-        bool stanzaSpecified = cfgOptionTest(cfgOptStanza);
-
-        if (stanzaSpecified)
-        {
-            stanzaList = strLstNew();
-            strLstAdd(stanzaList, cfgOptionStr(cfgOptStanza));
-        }
+        if (cfgOptionBool(cfgOptReport))
```

Comment:
If the idea is for the report to be based on the results of the check command... then it seems kind of odd to entirely skip the actual check command run when this option is enabled?  Or at least, that's what it looks like is happening here.

Other than that ... the actual report generation and such seems pretty reasonable.  I do wonder if the rendering of all the options and such might be something we want to support calling from places other than check, but I suppose we can move the code around in the future if that ends up being the case.

Reply:
Operative phrase here is `intended to eventually create a comprehensive report`. This started as a separate `support` command that I decided to merge into check instead. So no, it doesn't really do what is advertised, which is why it will remain internal for now. But since it touches stuff in the config module (a pretty fluid area) seems good to go ahead and get it in. Also, this way we can play with it and iterate.

Reply:
I'd at least make a note in the commit message and/or possibly in the comments around this that we currently aren't actually running 'check' as part of this, but that we will be in the future.

Reply:
OK, update commit message above. Also keep in mind this option is internal and not user-facing.